recipes/bloaty/all/patches/cmake_dependencies.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 08965ac..c05f5d7 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2,7 +2,6 @@ cmake_minimum_required(VERSION 3.5)
+ cmake_policy(SET CMP0048 NEW)
+ project (Bloaty VERSION 1.0)
+ project (Bloaty VERSION 1.1)
+-set(CMAKE_CXX_STANDARD 11)
+ 
+ # Options we define for users.
+ option(BLOATY_ENABLE_ASAN "Enable address sanitizer." OFF)
+@@ -10,6 +9,8 @@ option(BLOATY_ENABLE_UBSAN "Enable undefined behavior sanitizer." OFF)
+ option(BLOATY_ENABLE_CMAKETARGETS "Enable installing cmake target files." ON)
+ option(BLOATY_ENABLE_BUILDID "Enable build id." ON)
+ 
++find_package(absl REQUIRED)
++
+ if(UNIX)
+ find_package(PkgConfig)
+ if(${PKG_CONFIG_FOUND})
+@@ -89,7 +90,6 @@ endif(UNIX)
+ 
+ include_directories(.)
+ include_directories(src)
+-include_directories(third_party/abseil-cpp)
+ include_directories("${CMAKE_CURRENT_BINARY_DIR}/src")
+ 
+ # Baseline build flags.
+@@ -97,7 +97,6 @@ set(CMAKE_CXX_FLAGS "-std=c++11 -W -Wall -Wno-sign-compare")
+ set(CMAKE_CXX_FLAGS_DEBUG "-g1")
+ set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+ set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g1")
```

Comment:
Maybe you should also remove those flags (std=c++11 for sure because you have already removed CXX_STANDARD 11).